tests/launch.rs

```diff
@@ -1,13 +1,26 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use kvm_ioctls::Kvm;
+use kvm_bindings::{kvm_enable_cap, KVM_CAP_MAX_VCPUS, KVM_CAP_SPLIT_IRQCHIP};
 
 use tdx::launch::TdxVm;
 
 #[test]
 fn launch() {
     let kvm_fd = Kvm::new().unwrap();
     let tdx_vm = TdxVm::new(&kvm_fd).unwrap();
+
+    let mut cap: kvm_enable_cap = Default::default();
+    cap.cap = KVM_CAP_MAX_VCPUS;
+    cap.args[0] = 100;
+    tdx_vm.fd.enable_cap(&cap).unwrap();
+
+    cap.cap = KVM_CAP_SPLIT_IRQCHIP;
```

Comment:
Likewise, since we can't create vCPUs without `SPLIT_IRQCHIP`, this should also be called within `init_vm`?